commit db7d20e93b13ce985937a33d9137488ff2c7d2e1
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Mon Mar 13 12:30:37 2017 +0000

    thermal: Compatible strings for bcm2836, bcm2837
    
    The upstream dt-bindings documentation for bcm2835-thermal (which
    exists even though the driver isn't upstreamed) says to use
    dedicated compatible strings on bcm2836 and bcm2837, even though
    the downstream driver doesn't support them. The Pi2 DTB uses
    "brcm,bcm2836-thermal", so the driver doesn't load. The Pi3 DTB
    doesn't override the base value, but the arm64 Pi3 support uses "brcm,bcm2837-thermal".
    
    Solve the documentation problem by adding "brcm,bcm2836-thermal" and
    "brcm,bcm2837-thermal" as alternative compatible strings for the
    bcm2835-thermal driver.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/drivers/thermal/bcm2835-thermal.c b/drivers/thermal/bcm2835-thermal.c
index c63fb9f9d143..25b78c3eac15 100644
--- a/drivers/thermal/bcm2835-thermal.c
+++ b/drivers/thermal/bcm2835-thermal.c
@@ -89,6 +89,8 @@ static int bcm2835_thermal_remove(struct platform_device *pdev)
 
 static const struct of_device_id bcm2835_thermal_of_match_table[] = {
 	{ .compatible = "brcm,bcm2835-thermal", },
+	{ .compatible = "brcm,bcm2836-thermal", },
+	{ .compatible = "brcm,bcm2837-thermal", },
 	{},
 };
 MODULE_DEVICE_TABLE(of, bcm2835_thermal_of_match_table);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1129_db7d20e93b13ce985937a33d9137488ff2c7d2e1.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1129_db7d20e93b13ce985937a33d9137488ff2c7d2e1.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
